pkgs/development/interpreters/python/tests/test_python.py

```diff
@@ -35,6 +35,9 @@ def test_prefix(self):
     def test_site_prefix(self):
         self.assertTrue(sys.prefix in site.PREFIXES)
 
+        if IS_NIXENV:
+            self.assertTrue(ENV in site.PREFIXES)
```

Comment:
I think this is already handled. In `test_prefix` we already check if `sys.prefix == ENV`.

Reply:
You're right. I've removed this bit.